homeassistant/components/pandora/media_player.py

```diff
@@ -142,30 +149,24 @@ def turn_off(self) -> None:
         self._attr_state = MediaPlayerState.OFF
         self.schedule_update_ha_state()
 
-    def media_play(self) -> None:
+    async def async_media_play(self) -> None:
         """Send play command."""
-        self._send_pianobar_command(SERVICE_MEDIA_PLAY_PAUSE)
+        await self._send_pianobar_command(SERVICE_MEDIA_PLAY_PAUSE)
         self._attr_state = MediaPlayerState.PLAYING
         self.schedule_update_ha_state()
 
-    def media_pause(self) -> None:
+    async def async_media_pause(self) -> None:
         """Send pause command."""
-        self._send_pianobar_command(SERVICE_MEDIA_PLAY_PAUSE)
+        await self._send_pianobar_command(SERVICE_MEDIA_PLAY_PAUSE)
         self._attr_state = MediaPlayerState.PAUSED
         self.schedule_update_ha_state()
 
-    def media_next_track(self) -> None:
+    async def async_media_next_track(self) -> None:
         """Go to next track."""
-        self._send_pianobar_command(SERVICE_MEDIA_NEXT_TRACK)
+        await self._send_pianobar_command(SERVICE_MEDIA_NEXT_TRACK)
         self.schedule_update_ha_state()
 
-    @property
-    def media_title(self) -> str | None:
-        """Title of current playing media."""
-        self.update_playing_status()
-        return self._attr_media_title
```

Comment:
Why is this deleted?

Reply:
The method `update_playing_status` turned async, and I don't think it makes sense to await a coroutine for a property lookup (I don't even know if that is possible). The parent class already has a `@cached_property` `media_title` that only uses the last value of `_attr_media_title`, so that seemed good enough.

Reply:
But where do we set `_attr_media_title`?